filebeat/docs/command-line.asciidoc

```diff
@@ -1,9 +1,15 @@
 [[filebeat-command-line]]
 === Command Line Options
 
-Filebeat does not have any Filebeat-specific command line options. Instead, you
+Filebeat does only have one Filebeat-specific command line option. For all other options you
 configure the behaviour of Filebeat by specifying options in the configuration file.
 
+*`-once`*::
+When the `-once` flag is used, filebeat starts all configured harvesters and prospectors and runs
+each prospector until the harvesters are closed. It is recommended to use the flag in combination
+with `close_eof` so harvester directly close when the end of the file is reached.
```

Comment:
Maybe a little less formal would be good here: "If you set the `once` flag, you should also set `close_eof` so the harvester is closed when the end of the file is reached."